specification/nginx/resource-manager/NGINX.NGINXPLUS/stable/2023-09-01/swagger.json

```diff
@@ -0,0 +1,1579 @@
+        },
+        "display": {
+          "description": "The object that represents the operation.",
+          "$ref": "#/definitions/OperationDisplay"
+        },
+        "isDataAction": {
+          "description": "Indicates whether the operation is a data action",
+          "type": "boolean"
+        }
+      }
+    },
+    "OperationListResult": {
+      "description": "Result of GET request to list Nginx.NginxPlus operations.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of operations supported by the Nginx.NginxPlus provider.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OperationResult"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "URL to get the next set of operation list results if there are any.",
+          "type": "string"
+        }
+      }
+    },
+    "AnalysisCreate": {
+      "type": "object",
+      "description": "The request body for creating an analysis for an NGINX configuration.",
+      "properties": {
+        "config": {
+          "type": "object",
+          "properties": {
+            "rootFile": {
+              "type": "string",
+              "description": "The root file of the NGINX config file(s). It must match one of the files' filepath."
+            },
+            "files": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/NginxConfigurationFile"
+              },
+              "x-ms-identifiers": []
+            },
+            "protectedFiles": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/NginxConfigurationFile"
+              },
+              "x-ms-identifiers": []
+            },
+            "package": {
+              "$ref": "#/definitions/NginxConfigurationPackage"
+            }
+          }
+        }
+      },
+      "required": [
+        "config"
+      ]
+    },
+    "AnalysisResult": {
+      "type": "object",
+      "description": "The response body for an analysis request. Contains the status of the analysis and any errors.",
+      "required": [
+        "status"
+      ],
+      "properties": {
+        "status": {
+          "type": "string",
+          "description": "The status of the analysis."
+        },
+        "data": {
+          "type": "object",
+          "properties": {
+            "errors": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/AnalysisError"
+              }
+            }
+          }
+        }
+      }
+    },
+    "AnalysisError": {
+      "type": "object",
+      "description": "An error object found during the analysis of an NGINX configuration.",
+      "required": [
+        "directive",
+        "description",
+        "file",
+        "line",
+        "message",
+        "rule"
+      ],
+      "properties": {
+        "id": {
```

Comment:
>id [](http://example.com/codeflow?start=9&length=2)

Recommend against using property names the same as top-level properties in the standard ARM resource envelope. It's actually allowed in a POST payload, but I'd recommend errorId or similar instead of id.